Source/Objects/Misc Objects/InFluxDB/ORInFluxDBModel.m

```diff
@@ -547,19 +547,19 @@ - (void) processStatusLogLine:(NSNotification*)aNote
             }
 //        }
         }];
+
+    NSString* measurementName;
```

Comment:
The previous implementation was sending the userMessage twice to the statuslog measurement.